code/datums/uplink_item.dm

```diff
@@ -644,7 +644,7 @@ var/list/uplink_items = list()
 
 /datum/uplink_item/stealthy_weapons/sleepy_pen
 	name = "Sleepy Pen"
-	desc = "A syringe disguised as a functional pen. It's filled with a potent anaesthetic. \The pen holds two doses of the mixture. The pen can be refilled."
+	desc = "A syringe disguised as a functional pen. It's filled with a potent anaesthetic. \ The pen holds two doses of the mixture. The pen can be refilled."
```

Comment:
This breaks the `\The` text macro. I don't know why there was a text macro there in the first place, the backslash should be wholly unnecessary.